faust/sensors/prometheus.py

```diff
@@ -140,7 +153,7 @@ def _initialize_metrics(self) -> None:
             'topic_partition_offset_commited',
             'Offset commited per topic/partition',
             ['topic', 'partition'])
-        self.consumer_commit_latency = Summary(
+        self.consumer_commit_latency = Histogram(
             'consumer_commit_latency', 'Consumer commit latency in ms')
 
     def on_message_in(self, tp: TP, offset: int, message: Message) -> None:
```

Comment:
In line 191, if self.events_runtime is Event RuntTime deque([]), it throws an error because of the -1.

Reply:
A hotfix to suppress the error can be

```
        try:
            return self.events_runtime_latency.observe(self.secs_to_ms(self.events_runtime[-1]))
        except Exception as e:
            return self.events_runtime_latency.observe(self.secs_to_ms(0))
```

This is not a long term solution, but only a patch to get the code working